index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="logo black">
+          <a class="black" href="#">Logo</a>
+        </div>
```

Comment:
- [ ] **Required:** The original design doesn't use a logo on the navbar, please remove it from your project so it matches the figma design

Original figma design
![image](https://user-images.githubusercontent.com/50066876/236189825-d4fc7808-964e-4c28-a324-265215102f83.png)

Your design
![image](https://user-images.githubusercontent.com/50066876/236189908-02d1d8b8-c7ed-44ba-87c4-6b2490eb2a49.png)